tools/pcmsolver.py.in

```diff
@@ -217,94 +216,66 @@ def setup_keywords():
     # Cavity section
     cavity = getkw.Section('CAVITY', callback = verify_cavity)
     # Type of the cavity
-    # Valid values: GEPOL, WAVELET, TSLESS and RESTART
+    # Valid values: GEPOL and RESTART
     cavity.add_kw('TYPE',        'STR')
     # Name of the file containing data for restarting a cavity
     # Valid for: Restart cavity
     # Default: empty string
     cavity.add_kw('NPZFILE',     'STR', '')
-    # Name of the file containing the list of points for a wavelet cavity
-    # Valid for: Wavelet cavity
-    # Default: empty string
-    cavity.add_kw('DYADICFILE',     'STR', '')
-    # Patch level
-    # Valid for: Wavelet cavity
-    # Valid values: integer strictly greater than 0
-    # Default: 2
-    cavity.add_kw('PATCHLEVEL',  'INT', 2)
-    # Coarsity
-    # Valid for: Wavelet cavity
-    # Valid values: double in ]0.0, 1.0[
-    # Default: 0.5
-    cavity.add_kw('COARSITY',    'DBL', 0.5)
     # Average area (in au^2)
-    # Valid for: GePol and TsLess cavities
-    # Valid values: double strictly greater than 0.01 au^2
-    # Default: 0.3 au^2
+    # Valid for: GePol cavity
+    # Valid values: double strictly greater than 0.01 Angstrom^2
+    # Default: 0.3 Angstrom^2 (later converted to 1.07132... au^2) 
```

Comment:
The minimal input assumes atomic units as defaults, as such the default for areas really is 0.3 AU^2 We should maybe re-set the default to the AU^2 value of 0.3 Angstrom^2, instead of having this oblique comment in the script.